lib/puppet/type/base_dsc.rb

```diff
@@ -1,10 +1,7 @@
 # This is the base type for dsc.
 # Used to inherit the providers for the generated DSC resources
-require 'puppet_x/string'
-
 Puppet::Type.newtype(:base_dsc) do
 
   newparam(:name, :namevar => true ) do
   end
-
```

Comment:
Yeah, it's a good question, and what I originally started to do.  But I couldn't get that quite working given the way the types are defined:

https://github.com/puppetlabs/puppetlabs-dsc/blob/master/lib/puppet/type/dsc_file.rb#L6
https://github.com/puppetlabs/puppetlabs-dsc/blob/master/lib/puppet/type/dsc_file.rb#L8

The `munge_boolean` wasn't actually in the scope of the type.